packages/x-charts/src/context/SeriesContextProvider.tsx

```diff
@@ -16,6 +16,10 @@ import { ChartsColorPalette, blueberryTwilightPalette } from '../colorPalettes';
 
 export type SeriesContextProviderProps = {
   dataset?: DatasetType;
+  /**
+   * The array of series to display.
+   * Each type of series has it's own specificity. Please refer the appropriated docs page to know it.
```

Comment:
Could we link or at least mention the page name here? 🤔 
```suggestion
   * Each type of series has its own specificity. Please refer to the appropriate docs page to learn more about it.
```